lib/data/multipleBackendGateway.js

```diff
@@ -18,6 +18,16 @@ function createLogger(reqUids) {
 
 const clients = parseLC(config);
 
+function _createAwsKey(requestBucketName, requestObjectKey,
+    partNumber, uploadId, bucketMatch) {
+    if (bucketMatch) {
```

Comment:
Should not it be: if bucketMatch => real AWS key starting with `requestBucketName`?